ajax/libs/vue-functional-calendar/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "version": "2.8.69",
+  "description": "Lightweight, high-performance calendar component based on Vue.js",
+  "keywords": [
+     "vue",
```

```suggestion
    "vue",
```
2 space indentation, please.